pkg/pillar/agentlog/agentlog.go

```diff
@@ -27,7 +27,8 @@ const (
 	stackFile      = "reboot-stack"
 	rebootImage    = "reboot-image"
 	bootReasonFile = "boot-reason"
-	maxReadSize    = 16384 // From files in /persist
+	readSize16k    = 16 << 10  // From files in /persist
+	readSize128k   = 128 << 10 // Kernel dmesg
```

Comment:
I can see that for some reason we use different `CONFIG_LOG_BUF_SHIFT` for [aarch64](https://github.com/lf-edge/eve/blob/4c1d24d07206db45bd70afb7e4b1597fbeafb690/pkg/kernel/kernel_config-5.10.x-aarch64#L126): CONFIG_LOG_BUF_SHIFT=18. WDYT, should we sync `CONFIG_LOG_BUF_SHIFT`? Cannot find any good recommendations...

Reply:
256k - that's a lot. Better to be in sync on all archs, of course, even though we don't collect kdumps on arm now. Good catch.

Reply:
Or we put readSize256k - and forget about it :)

Reply:
this is userspace app, so who cares 128k or 256k afterall for a buffer.

Reply:
Yeah the failures we ran into was when we were reading a file which had grown above 1Gb. So 1Mb is fine as well.